pandas/core/tools/datetimes.py

```diff
@@ -627,7 +627,7 @@ def to_datetime(
         - If True, require an exact format match.
         - If False, allow the format to match anywhere in the target string.
 
-    unit : str, default 'ns'
+    unit : str, default None
```

Comment:
I think this was OK to keep as it was since the default value unit would be `ns` - can you revert that?

Reply:
@WillAyd Thanks for the comment! I reverted it.